### ISSUE TITLE
Can't login to Google with two-factor authentication

### ISSUE THREAD

**Issue by sskelton90** [mauimauer/portify#32]
I have tried logging in with my Google account using my correct username and password but I get an error that login failed. I use two-factor authentication on my Google account so I believe it may be related to this.

**Comment by aericson** [comment]
bypass using this: https://support.google.com/accounts/answer/185833

**Comment by emailrob** [comment]
Yep, an application specific password works. Still, it crashes after the playlist screen for me so will keep trying ;)

**Comment by dpc** [comment]
@aericson: I don't think it's a bypass. It's a official method of login in for 3rd-party applications, I think.

This bug should really be changed so that Portify includes information about app-specific passwords, becoause some people might not be aware of it.

**Comment by aericson** [comment]
@dpc that's what I meant, bypass for portify as it should tell user to use app-specific instead
